elasticapm/instrumentation/packages/sqlite.py

```diff
@@ -52,12 +53,13 @@ class SQLiteConnectionProxy(ConnectionProxy):
 
     def _trace_sql(self, method, sql, params):
         signature = extract_signature(sql)
+        sql_string = keyword_field(sql)
```

Comment:
IIRC, some other agents limit the size of SQL queries to 10000 characters (Java agent for sure, not sure about others). Maybe it makes sense to align.